module/network/RobotCommunication/src/RobotCommunication.cpp

```diff
@@ -0,0 +1,53 @@
+#include "RobotCommunication.hpp"
+#include "extension/Configuration.hpp"
+
+#include "message/input/GameState"
+#include "message/localisation/FilteredBall.hpp"
+
+namespace module::network {
+
+    using extension::Configuration;
+
+    RobotCommunication::RobotCommunication(std::unique_ptr<NUClear::Environment> environment)
+        : Reactor(std::move(environment)) {
+
+        on<Configuration>("RobotCommunication.yaml").then([this](const Configuration& config) {
+            // Use configuration here from file RobotCommunication.yaml
+            this->log_level = config["log_level"].as<NUClear::LogLevel>();
+            port            = config["port"].as<uint>();
```

Comment:
you've not declared this variable anywhere - there will be a compile error. I've made another comment where it should be declared within the class.